### PR TITLE
CM-316: Create a checkbox "include unpaid" on payroll creation page

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/CM-316

### CODE REVIEW

File: src/pages/payroll/PayrollHeadPanel.js [hunk 1]
```diff
@@ -181,6 +181,18 @@ class PayrollHeadPanel extends FormPanel {
               readOnly={readOnly}
             />
           </Grid>
+          <Grid item className={classes.item}>
+            <FormControlLabel
+              label={formatMessage(intl, 'payroll', 'payroll.includedUnpaid')}
+              control={(
+                <Checkbox
+                  checked={!!payroll && !!payroll.includedUnpaid && payroll.includedUnpaid}
```
Comment (olewandowski1) [inline src/pages/payroll/PayrollHeadPanel.js]: I think it can be shortened to:
`
checked={!!payroll.includedUnpaid}
`

**Review by olewandowski1 — COMMENTED** [review]
@sniedzielski I left one comment. Could you take a look on it?

File: src/pages/payroll/PayrollHeadPanel.js (hunk 1 above)
Comment (sniedzielski) [reply src/pages/payroll/PayrollHeadPanel.js]: ok changed and it works too

**Comment by sniedzielski** [comment]
@olewandowski1 I fixed your comment. Please take a look again.

**Review by olewandowski1 — APPROVED** [review]
@sniedzielski LGTM, thanks!

**Review by dborowiecki — APPROVED** [review]
Please merge this PR after https://github.com/openimis/openimis-be-payroll_py/pull/10#pullrequestreview-1637192249 is merged.

**Pull request merged**